sdk/messaging/azservicebus/log.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azservicebus
+
+import "github.com/Azure/azure-sdk-for-go/sdk/azcore/log"
+
+const (
+	// EventConn is used whenever we create a connection or any links (ie: receivers, senders).
+	EventConn log.Event = "azsb.Conn"
```

Comment:
This brings up a good question about how SDKs should name their events.

CC @JeffreyRichter

Reply:
Since only the Azure SDK team can define these strings and we mostly need them for HTTP and some for AMQP, I think the event strings should be geared towards wire protocol more than towards service. Could service bus over AMQP use the same log event strings, for example (I'd hope so).

Reply:
These constants are actually just SB/AMQP (except for the last one, `EventAdmin`, which covers the HTTP/ATOM client).

I'm guessing I can delete that one _soon_ once I transition over to using the normal azcore HTTP pipeline.

Reply:
@JeffreyRichter , @jhendrixMSFT : Were we okay with the prefixing of the sb specific events?

Reply:
We need to distill a general pattern.  None of our other events have a prefix, so if we want to do this, then we should be consistent.
I'm on the fence about it.  I see the value in preventing collisions in event names, but it reads weird to me.